.github/workflows/ci.yml

```diff
@@ -297,35 +296,18 @@ jobs:
       SDL2_FRAMEWORK: build/SDL2.framework
     steps:
     - uses: actions/checkout@v3
-    - name: Restore cached game binary
-      id: cache-artifact
-      uses: actions/cache@v3
-      with:
-        path: ${{ env.ARTIFACT }}
-        key: ${{ matrix.os }}-artifacts-${{ hashFiles('source/**', '.github/workflows/ci.yml', 'EndlessSky.xcodeproj/**', 'utils/set_dylibs_rpath.sh', 'utils/fetch_sdl2_framework.sh') }}
```

Comment:
If this job no longer caches binaries, it should only build when the source changed, right? Just like the [cmake job](https://github.com/endless-sky/endless-sky/blob/73bef74a26b1080ddf173062804794292cfa9ed0/.github/workflows/ci.yml#L522-L523)